tensorflow/core/profiler/convert/op_stats_to_overview_page.cc

```diff
@@ -287,15 +287,16 @@ std::string TfFunctionRecommendationHtml(const TfFunctionDb& tf_function_db) {
   // Sorts candidates in descending order of expensive_call_percent.
   absl::c_sort(candidates, cmp);
   std::string expensive_functions = "";
-  auto num_functions_shown = std::min(
+  const int64 num_functions_shown = std::min(
       static_cast<decltype(candidates)::size_type>(3), candidates.size());
```

Comment:
This should be kept as `auto` since it is supposed to be the same as the `decltype...`